lib/generator.js

```diff
@@ -893,6 +893,9 @@
      *     Note that this option *cannot* be used with an inputRect/outputRect scaling. If inputRect/outputRect
      *     is set, this setting will be ignored and the pixels will not be cropped to document bounds.
      *     (Default: false)
+     * @param {?integer} settings.maxDimension: This is the maximal dimension of pixmap that can be returned
```

Comment:
the `integer`/`number` choice isn't really clear as there are both variants used in `generator.json`. I changed this to `number` and we could maybe do that for all the other `integers`?